tests/test_gpg.py

```diff
@@ -52,6 +54,19 @@ def test_gpg_write_reveal(self):
         revealed = REVEALER.reveal_raw_file(file_with_secret_tags)
         self.assertEqual("I am a file with a super secret value", revealed)
 
+    def test_gpg_write_embedded_reveal(self):
+        "write and compile embedded secret, confirm secret file exists, reveal and compare content"
+        tag = "?{gpg:secret/sauce}"
+        REF_CONTROLLER_EMBEDDED[tag] = GPGSecret("super secret value", [{"fingerprint": KEY.fingerprint}])
+        self.assertTrue(os.path.isfile(os.path.join(REFS_HOME, "secret/sauce")))
+        ref_obj = REF_CONTROLLER_EMBEDDED[tag]
+
+        file_with_secret_tags = tempfile.mktemp()
+        with open(file_with_secret_tags, "w") as fp:
+            fp.write("I am a file with a {}".format(ref_obj.compile()))
```

Comment:
"with a tag {}" ?

Reply:
this will actually test the value, so I think "tag" is not needed here